src/main/java/org/junitpioneer/internal/PropertyUtils.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2016-2023 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+package org.junitpioneer.internal;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Utils to read a property file.
+ * */
+public final class PropertyUtils {
```

Comment:
This is how Jupiter does it, which allows giving precedence to e.g. JVM args:

* https://junit.org/junit5/docs/current/user-guide/#running-tests-config-params
* https://github.com/junit-team/junit5/blob/3daa2c5c69c81ab1f807b39dc1077c885351538e/junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/LauncherConfigurationParameters.java
* https://github.com/junit-team/junit5/blob/3daa2c5c69c81ab1f807b39dc1077c885351538e/junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/LauncherConfigurationParameters.java